<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BSTInterface</class>
 <widget class="QWidget" name="BSTInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestor BST</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="topLayout">
     <item>
      <widget class="QPushButton" name="btnCargar">
       <property name="text">
        <string>Cargar files.txt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblExt">
       <property name="text">
        <string>Buscar extensi√≥n:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="entradaExt"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnBuscar">
       <property name="text">
        <string>Buscar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="recorridoLayout">
     <item>
      <widget class="QPushButton" name="btnInorden">
       <property name="text">
        <string>Inorden</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPreorden">
       <property name="text">
        <string>Preorden</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPostorden">
       <property name="text">
        <string>Postorden</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textResultado">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnExportar">
     <property name="text">
      <string>Exportar CSV</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
